v0/util/index.js

```diff
@@ -1172,6 +1172,43 @@ function isCdkDestination(event) {
   );
 }
 
+/**
+ * This function helps to remove any undefined value from dynamicForm, dynamicCustomForm
+ * and dynamicSelectForm array.
+ *
+ * example:
+ * input: "eventsToEvents": [
+                    {
+                        "from": "spin_result",
+                        "to": "Schedule"
+                    },
+                    {
+                        "to": "Schedule"
+                    }
+                ]
+ * output: "eventsToEvents": [
+                    {
+                        "from": "spin_result",
+                        "to": "Schedule"
+                    }
+                ]
+
+ * @param {*} attributeArray -> This is the config output (array) from dynamicForm/dynamicCustomForm/dynamicSelectForm element.
+ * @param {*} keyLeft -> This is the name of the leftKey, in general it is 'from'.
+ * @param {*} keyRight -> This is the name of the rightKey, in general it is 'to'.
+ * @returns
+ */
+function validateDynamicFormConfig(attributeArray, keyLeft, keyRight) {
```

Comment:
As this function cleans and returns the config rather than validating it, we should name it better.
```suggestion
function getValidDynamicFormConfig(attributeArray, keyLeft, keyRight) {
```